zulipterminal/ui_tools/views.py

```diff
@@ -882,11 +888,9 @@ def topics_view(self, stream_button: Any) -> Any:
         self.view.topic_w = TopicsView(topics_btn_list, self.view, stream_button)
         w = urwid.LineBox(
             self.view.topic_w,
-            title="Topics",
-            title_attr="column_title",
-            tlcorner=COLUMN_TITLE_BAR_LINE,
-            tline=COLUMN_TITLE_BAR_LINE,
-            trcorner=COLUMN_TITLE_BAR_LINE,
```

Comment:
This looks independent of this particular commit, and maybe belongs in the previous commit?

Also, this change essentially sets all the borders to be empty?